### PR TITLE
style: update variables to style dictionary

### PR DESCRIPTION
**Pull request by ju-Skinner**
# Description
Updates the following components to use the new tokens from Token Studio

* Divider
* Input
* Link
* Progress
* Radio

Fixes #(issue)
https://kajabi.atlassian.net/browse/DSS-566
https://kajabi.atlassian.net/browse/DSS-569
https://kajabi.atlassian.net/browse/DSS-570
https://kajabi.atlassian.net/browse/DSS-571
https://kajabi.atlassian.net/browse/DSS-572

## Note
There is an unrelated issue discovered with Progress, that I will be addressing. TL;DR Progress doesn't show up due to the canvas display property set to `inline-flex`.

## Type of change

Please delete options that are not relevant.
If your type of change is not present, add that option.

- [x] Style changes

# How Has This Been Tested?
- [x] tested manually
- [x] Storybook

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼

File: libs/core/src/components/pds-divider/pds-divider.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  --offset-md: var(--pine-spacing-md);
-  --offset-lg: var(--pine-spacing-lg);
-  --offset-xxl: var(--pine-spacing-xxl);
+  --color-background: var(--pine-color-grey-400);
+  --offset-xxs: var(--pine-spacing-050);
+  --offset-xs: var(--pine-spacing-100);
+  --offset-sm: var(--pine-spacing-200);
+  --offset-md: var(--pine-spacing-300);
+  --offset-lg: var(--pine-spacing-450);
+  --offset-xxl: var(--pine-spacing-800);
```
Comment (anechol) [inline libs/core/src/components/pds-divider/pds-divider.scss]: How do you feel about prepending `spacing` as the category here?

File: libs/core/src/components/pds-divider/pds-divider.scss [hunk 2]
```diff
@@ -1,11 +1,11 @@
 :host {
-  --background-color: var(--pine-color-neutral-grey-400);
-  --offset-xxs: var(--pine-spacing-xxs);
-  --offset-xs: var(--pine-spacing-xs);
-  --offset-sm: var(--pine-spacing-sm);
-  --offset-md: var(--pine-spacing-md);
-  --offset-lg: var(--pine-spacing-lg);
-  --offset-xxl: var(--pine-spacing-xxl);
+  --color-background: var(--pine-color-grey-400);
```
Comment (anechol) [inline libs/core/src/components/pds-divider/pds-divider.scss]: Do you feel it's necessary to add `default` here at the end?

File: libs/core/src/components/pds-input/pds-input.scss [hunk 3]
```diff
@@ -1,33 +1,30 @@
 :host {
-  --background: var(--pine-color-base-white);
-  --background-disabled: var(--pine-color-neutral-grey-100);
+  --border-color-default: var(--pine-color-grey-400);
+  --border-color-hover: var(--pine-color-grey-500);
```
Comment (anechol) [inline libs/core/src/components/pds-input/pds-input.scss]: Do you think these should live in the `color` category?

File: libs/core/src/components/pds-input/pds-input.scss [hunk 4]
```diff
@@ -1,33 +1,30 @@
...
 
-  --color: var(--pine-color-neutral-charcoal-500);
+  --color: var(-pine-color-charcoal-500);
```
Comment (anechol) [inline libs/core/src/components/pds-input/pds-input.scss]: How about adding `text` here to provide the context and usage at a glance?

File: libs/core/src/components/pds-input/pds-input.scss [hunk 5]
```diff
@@ -1,33 +1,30 @@
...
-  --field-line-height: var(--pine-line-height-md);
-  --field-padding-block: var(--pine-spacing-xs);
-  --field-padding-inline: var(--pine-spacing-sm);
-
-  --helper-message-font-size: var(--pine-font-size-body);
-  --helper-message-font-weight: var(--pine-font-weight-regular);
-  --helper-message-line-height: var(--pine-line-height-sm);
-  --helper-message-margin-top: var(--pine-spacing-xs);
+  --font-label-size: var(--pine-font-size-100);
+  --font-label-weight: var(--pine-font-weight-medium);
```
Comment (anechol) [inline libs/core/src/components/pds-input/pds-input.scss]: Should these be `font-size` and `font-weight`?

File: libs/core/src/components/pds-input/pds-input.scss [hunk 6]
```diff
@@ -1,33 +1,30 @@
...
+  --font-weight-input-field: var(--pine-font-weight-normal);
+
+  --line-height-label: var(--pine-line-height-150);
+  --line-height-helper-message: var(--pine-line-height-125);
+  --line-height-input-field: var(--pine-line-height-150);
+
+  --spacing-field-padding-block: var(--pine-spacing-100);
+  --spacing-input-field-padding-inline: var(--pine-spacing-200);
+  --spacing-label-margin-block-end: var(--pine-spacing-100);
+  --spacing-margin-top-helper-message: var(--pine-spacing-100);
```
Comment (anechol) [inline libs/core/src/components/pds-input/pds-input.scss]: 🔥 🔥

File: libs/core/src/components/pds-link/pds-link.scss [hunk 7]
```diff
@@ -1,28 +1,31 @@
...
-  --box-shadow-focus: inset 0 0 0 2px var(--pine-color-primary-200);
-  --color: var(--pine-color-neutral-charcoal-400);
-  --font-size: var(--pine-font-size-body);
-  --font-size-sm: var(--pine-font-size-body-sm);
-  --font-size-xs: var(--pine-font-size-body-xs);
+  --border-outline: 4px solid var(--pine-color-blue-200);
+  --border-radius: var(--pine-border-radius-075);
+  --box-shadow-focus: inset 0 0 0 2px var(--pine-color-blue-200);
+
+  --color-text: var(--pine-color-charcoal-400);
```
Comment (anechol) [inline libs/core/src/components/pds-link/pds-link.scss]: Same as some others above: should we add `default` to the end here?

File: libs/core/src/components/pds-progress/pds-progress.scss [hunk 8]
```diff
@@ -1,6 +1,6 @@
 :host {
-  --progress-fill-color: var(--pine-color-primary-300);
-  --height-progress-bar: 8px;
+  --color-progress-fill: var(--pine-color-blue-300);
+  --sizing-progress-bar-height: 8px;
```
Comment (anechol) [inline libs/core/src/components/pds-progress/pds-progress.scss:3]: 🔥

File: libs/core/src/components/pds-radio/pds-radio.scss [hunk 9]
```diff
@@ -1,22 +1,25 @@
...
+  --border-interactive: var(--pine-border-interactive-default);
   --border-radius: 50%;
   --box-shadow-focus: 0 0 0 2px var(--color-focus);
   --box-shadow-focus-error: 0 0 0 2px var(--color-invalid-focus);
-  --color: var(--pine-color-neutral-charcoal-200);
-  --color-checked: var(--pine-color-neutral-charcoal-500);
-  --color-disabled: var(--pine-color-neutral-grey-500);
-  --color-focus: var(--pine-color-primary-200);
+
+  --check-size: 6px;
```
Comment (anechol) [inline libs/core/src/components/pds-radio/pds-radio.scss]: Can we add the category here?

**Review by anechol — CHANGES_REQUESTED** [review]
Awesome job getting these updated! Left a handful of small nitpicky comments over adding categories and context around usage within the names. There are more spots to update than were called out, but I will let you determine what should be updated for now. If you need any assistance or have questions let me know.

File: libs/core/src/components/pds-divider/pds-divider.scss (hunk 1 above)
Comment (ju-Skinner) [reply libs/core/src/components/pds-divider/pds-divider.scss]: agreed. This one was done before our conversation. LOL

File: libs/core/src/components/pds-radio/pds-radio.scss (hunk 9 above)
Comment (ju-Skinner) [reply libs/core/src/components/pds-radio/pds-radio.scss]: This didn't register that it should be `sizing-check-size` until you said something right now. Thanks for calling it out.

File: libs/core/src/components/pds-radio/pds-radio.scss [hunk 10]
```diff
@@ -1,22 +1,26 @@
...
+  --border-interactive: var(--pine-border-interactive-default);
   --border-radius: 50%;
   --box-shadow-focus: 0 0 0 2px var(--color-focus);
   --box-shadow-focus-error: 0 0 0 2px var(--color-invalid-focus);
-  --color: var(--pine-color-neutral-charcoal-200);
-  --color-checked: var(--pine-color-neutral-charcoal-500);
-  --color-disabled: var(--pine-color-neutral-grey-500);
-  --color-focus: var(--pine-color-primary-200);
+
+  --color: var(--pine-color-charcoal-200);
```
Comment (anechol) [inline libs/core/src/components/pds-radio/pds-radio.scss:7]: Would this be `color-text-default`?

File: libs/core/src/components/pds-input/pds-input.scss [hunk 11]
```diff
@@ -1,33 +1,30 @@
 :host {
-  --background: var(--pine-color-base-white);
-  --background-disabled: var(--pine-color-neutral-grey-100);
-
-  --color: var(--pine-color-neutral-charcoal-500);
+  --box-shadow-focus: 0 0 0 2px var(--pine-color-blue-200);
+  --box-shadow-focus-error: 0 0 0 2px var(--pine-color-red-200);
+
+  --color-text: var(-pine-color-charcoal-500);
```
Comment (anechol) [inline libs/core/src/components/pds-input/pds-input.scss]: Should we add default here as well?

File: libs/core/src/components/pds-input/pds-input.scss [hunk 12]
```diff
@@ -1,33 +1,30 @@
...
+  --color-background-disabled: var(--pine-color-grey-100);
+  --color-border-default: var(--pine-color-grey-400);
+  --color-border-hover: var(--pine-color-grey-500);
+  --color-disabled: var(--pine-color-grey-500);
   --color-error: var(--pine-color-red-300);
```
Comment (anechol) [inline libs/core/src/components/pds-input/pds-input.scss]: Are these two related to text color?

File: libs/core/src/components/pds-input/pds-input.scss [hunk 13]
```diff
@@ -1,33 +1,30 @@
...
-  --color-placeholder: var(--pine-color-neutral-charcoal-200);
-  --color-disabled: var(--pine-color-neutral-grey-500);
-
-  --border-color-default: var(--pine-color-neutral-grey-400);
-  --border-color-hover: var(--pine-color-neutral-grey-500);
-
-  --focus-box-shadow: 0 0 0 2px var(--pine-color-primary-200);
-  --focus-box-shadow-error: 0 0 0 2px var(--pine-color-red-200);
+  --color-placeholder: var(--pine-color-charcoal-200);
```
Comment (anechol) [inline libs/core/src/components/pds-input/pds-input.scss]: I could see adding `text` here being helpful. What do you think?

**Review by anechol — CHANGES_REQUESTED** [review]
A few more things I think could be addressed, but we're getting closer. Let me know what you think.

File: libs/core/src/components/pds-radio/pds-radio.scss (hunk 10 above)
Comment (ju-Skinner) [reply libs/core/src/components/pds-radio/pds-radio.scss:7]: @anechol This one is technically not being used anywhere.

File: libs/core/src/components/pds-input/pds-input.scss (hunk 11 above)
Comment (ju-Skinner) [reply libs/core/src/components/pds-input/pds-input.scss]: same as above...not being used anywhere either

Comment (anechol) [reply libs/core/src/components/pds-input/pds-input.scss]: All good!

**Review by anechol — APPROVED** [review]

**Comment by anechol** [comment]
Nice job on this!
